graphql/schema/types/distro.graphql

```diff
@@ -291,6 +292,7 @@ type Distro {
   note: String!
   plannerSettings: PlannerSettings!
   provider: Provider!
+  providerAccount: String
```

Comment:
It makes sense that this field is optional for `DistroInput`, but can it be required for `Distro` and just return an empty string? Pretty much all fields on this type are required to make frontend implementation easier

e.g. `note` and `imageId` are required on this type but are actually allowed to be empty if you check on prod

Reply:
That makes sense